mithril-common/src/messages/cardano_transaction_snapshot.rs

```diff
@@ -60,14 +63,11 @@ mod tests {
 
     // Test the retro compatibility with possible future upgrades.
     #[test]
-    fn test_v1() {
+    fn test_v2() {
```

Comment:
We can probably leave it as `test_v1` as the feature is still unstable:
```suggestion
    fn test_v1() {
```